tests/test_scene_rendering/test_cli_flags.py

```diff
@@ -91,6 +91,31 @@ def test_s_flag_no_animations(tmp_path, manim_cfg_file, simple_scenes_path):
     assert not is_empty, "running manim with -s flag did not render an image"
 
 
+@pytest.mark.slow
+def test_no_s_flag_no_animations(tmp_path, manim_cfg_file, simple_scenes_path):
+    scene_name = "NoAnimations"
+    command = [
+        "python",
+        "-m",
+        "manim",
+        simple_scenes_path,
+        scene_name,
+        "-ql",
+        "--media_dir",
+        str(tmp_path),
+    ]
+    out, err, exit_code = capture(command)
+    assert exit_code == 0, err
+
+    # As of right now, video folder would still be created, as folder creation is done prior to rendering
+
+    # exists = (tmp_path / "videos").exists()
+    # assert not exists, "running manim without -s flag rendered a video"
```

Comment:
you could change this to check if the video files were actually created in ("videos" / "simple_scenes" / "480p15" / "partial_movie_files" / scene_name) by checking if its empty similar to below

And the error could read similarly to below "running manim on scene with no animation scene without -s flag did rendered a video"